tests/test_query.py

```diff
@@ -248,10 +254,21 @@ def test_restconf_query_depth_2_trunk():
     assert response.json() == json.loads("""
 {
     "settings": {
+        "complextime": {},
         "debug": "enable",
+        "description": {},
```

Comment:
these are leaves not containers ... hence should not return an empty json object